File: src/main/java/com/scrumwall/service/item/ItemService.scala
package com.scrumwall.service.item

import com.scrumwall.domain.Item
import java.util.List

trait ItemService extends BaseService{
	
  def get(id: Int) : Item
  
  def save(item: Item) : Item
  
  def getForSprint(sprintId: Int) : List[Item]
  
  def getItems(columnId: Int) : List[Item]

  def remove(id: Int)
  
}

File: src/main/java/com/scrumwall/gwt/client/ItemService.java
package com.scrumwall.gwt.client;

import java.util.List;

import com.google.gwt.user.client.rpc.RemoteService;
import com.google.gwt.user.client.rpc.RemoteServiceRelativePath;
import com.scrumwall.gwt.shared.ItemDTO;

@RemoteServiceRelativePath("service/itemService")
public interface ItemService extends RemoteService {
	List<ItemDTO> getForColumn(int columnId);
	void save(ItemDTO item);
}
